src/App.tsx

```diff
@@ -1,8 +1,25 @@
-import React from 'react';
+import { useState } from 'react';
+
+import Modal from './components/Modal';
+import Header from './components/Header';
+import Input from './components/Input';
+import Todos from './components/Todos';
 import './App.css';
 
 function App() {
-  return <div></div>;
+  const [createTodoModalIsShown, setCreateTodoModalIsShown] = useState(false);
+
+  const showTodoModalHandler = () => setCreateTodoModalIsShown(true);
```

Comment:
You can create one function and pass true/false, or make setCreateTodoModalIsShown(!createTodoModalIsShown) in function